src/uct/ib/dc/base/dc_ep.h

```diff
@@ -156,7 +156,10 @@ static inline void uct_dc_iface_dci_put_dcs(uct_dc_iface_t *iface, uint8_t dci)
     ucs_assert(iface->tx.stack_top > 0);
 
     if (uct_dc_iface_dci_has_outstanding(iface, dci)) {
-        ucs_assert(ep != NULL);
+        if (ep == NULL) {
+            /* The EP was destroyed after flush cancel */
```

Comment:
can we add some assertion to verify that?

Reply:
Done